src/ast/mod.rs

```diff
@@ -1536,6 +1536,10 @@ pub enum Statement {
         replace_into: bool,
         /// Only for mysql
         priority: Option<MysqlInsertPriority>,
+        /// Only for mysql
```

Comment:
Could you please add an example to this docstring with an example of the supported syntax?

Reply:
Also, it appears that mysql calls these fields `row_alias` and `col_alias`
<img width="551" alt="Screenshot 2024-02-28 at 5 36 08 PM" src="https://github.com/sqlparser-rs/sqlparser-rs/assets/490673/20325dd2-4be6-4fc5-9bc4-5f965354e19b">

https://dev.mysql.com/doc/refman/8.0/en/insert.html

What do you think about using the same names?

Reply:
It is a good idea. The field names were changed.